### PR TITLE
[DE] Fix issues in `History of osu!/April Fools`

### PR DESCRIPTION
**Pull request by The-Last-Cookie**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/History_of_osu!/April_Fools/de.md [hunk 1]
```diff
@@ -131,15 +131,15 @@ Das Update wurde von den Spielern allgemein gut angenommen und eine tatsächlich
...
+In Bezug auf die Art und Weise, wie Spieler die Oculus Rift als Eingabemethode nutzen können, wird im Newspost beschrieben, dass "der Spieler den Cursor in der Ich-Perspektive auf seinem Weg von und zu Hit-Objekten auf dem Bildschirm erlebt (…)" und dass Hit-Objekte getroffen werden, "(…) indem der Spieler vokalähnliche Laute mit seinem Mund erzeugt".[^translated] Es wurden keine tatsächlichen Änderungen im Spiel vorgenommen, die mit der Verwendung von Oculus Rift oder einem ähnlichen [Gerät mit VR-Unterstützung (Virtuelle Realität)](https://de.wikipedia.org/wiki/Virtuelle_Realit%C3%A4t) zur Steuerung von osu! zusammenhängen.[^osu-vr-news]
```
Comment (The-Last-Cookie) [inline wiki/History_of_osu!/April_Fools/de.md:134]: cc @FreezyLemon You mentioned that you prefer it to be explicitly mentioned when citations are translated. Please take a look at this if this is okay for you.

File: wiki/History_of_osu!/April_Fools/de.md [hunk 2]
```diff
@@ -131,15 +131,15 @@ Das Update wurde von den Spielern allgemein gut angenommen und eine tatsächlich
...
 
 *Hinweis: McOsu wird getrennt entwickelt und steht in keiner direkten Verbindung mit osu! oder ppy Pty Ltd.*
```
Comment (The-Last-Cookie) [inline wiki/History_of_osu!/April_Fools/de.md:136]: I always found it weird that this line is displayed before McOsu is actually introduced in the paragraph below. Maybe this should be moved down at some point? 🤔

(out of scope here)

File: wiki/History_of_osu!/April_Fools/de.md [hunk 3]
```diff
@@ -131,15 +131,15 @@ Das Update wurde von den Spielern allgemein gut angenommen und eine tatsächlich
...
+Obwohl die offiziellen osu!-Entwickler keine Intention haben/hatten, VR-Unterstützung anzubieten, hat die Idee von osu! in VR bei einigen Fans Interesse geweckt. Aus diesem Interesse entstand kurz darauf ein inoffizielles Fanprojekt, welches das Ziel hatte, einen kostenlosen und quelloffenen Client zu kreieren, um [osu!-Beatmaps](/wiki/Beatmap) üben zu können, mit weiteren Funktionen und [Spielmodifikationen](/wiki/Gameplay/Game_modifier), einschließlich der Möglichkeit, in VR zu spielen. Unter dem Namen [McOsu](https://store.steampowered.com/app/607260/McOsu) wurde das Projekt am 20. März 2017 fertiggestellt und auf [Steam](https://de.wikipedia.org/wiki/Steam) veröffentlicht.[^osu-vr-reddit][^osu-vr-yt][^osu-vr-gameskinny][^osu-vr-mcosu]
```
Comment (FreezyLemon) [inline wiki/History_of_osu!/April_Fools/de.md:138]: `... üben zu können, mit weiteren Funktionen und [Spielmodifikationen](/wiki/Gameplay/Game_modifier), einschließlich der Möglichkeit, in VR zu spielen.`

This is pretty bad, and the sentence is too convoluted. Can you reword or maybe split this sentence while you're at it?

Comment (The-Last-Cookie) [reply wiki/History_of_osu!/April_Fools/de.md:138]:
```suggestion
Obwohl die offiziellen osu!-Entwickler keine Intention haben/hatten, VR-Unterstützung anzubieten, hat die Idee von osu! in VR bei einigen Fans Interesse geweckt. Aus diesem Interesse entstand kurz darauf ein inoffizielles Fanprojekt, welches das Ziel hatte, einen kostenlosen und quelloffenen Client zu kreieren, um [osu!-Beatmaps](/wiki/Beatmap) üben zu können, mit weiteren Funktionen und [Spielmodifikationen](/wiki/Gameplay/Game_modifier). Außerdem gab es die Möglichkeit, in VR zu spielen. Unter dem Namen [McOsu](https://store.steampowered.com/app/607260/McOsu) wurde das Projekt am 20. März 2017 fertiggestellt und auf [Steam](https://de.wikipedia.org/wiki/Steam) veröffentlicht.[^osu-vr-reddit][^osu-vr-yt][^osu-vr-gameskinny][^osu-vr-mcosu]
```

File: wiki/History_of_osu!/April_Fools/de.md (hunk 1 above)
Comment (FreezyLemon) [reply wiki/History_of_osu!/April_Fools/de.md:134]: it's good

**Review by FreezyLemon — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**